contracts/datastore/DataStore.sol

```diff
@@ -16,13 +16,17 @@ contract DataStore is DataStoreStorage, IDataStore {
     event SecurityTokenChanged(address indexed _oldSecurityToken, address indexed _newSecurityToken);
 
     modifier onlyAuthorized() {
-        bool isOwner = msg.sender == IOwnable(address(securityToken)).owner();
-        require(isOwner ||
-            securityToken.isModule(msg.sender, DATA_KEY) ||
-            securityToken.checkPermission(msg.sender, address(this), MANAGEDATA),
+        _isAuthorized();
```

Comment:
Any reason not to just replace the whole modifier with a call to _isAuthorized()? Not convinced that modifiers are easier to read than a function call...

Reply:
The only reason to keep as it is to keep functions readable.